src-plugins/reformat/medCropToolBox.cpp

```diff
@@ -160,11 +160,13 @@ void medCropToolBox::updateView()
     medTabbedViewContainers * containers = getWorkspace()->stackedViewContainers();
     QList<medViewContainer*> containersInTabSelected =  containers->containersInTab(containers->currentIndex());
     medAbstractLayeredView* view = NULL;
+    medViewContainer* currentContainer = NULL;
     for(int i=0;i<containersInTabSelected.length();i++)
     {
         if (containersInTabSelected[i]->isSelected())
         {
             view = qobject_cast<medAbstractLayeredView*>(containersInTabSelected[i]->view());
+            currentContainer = containersInTabSelected[i];
```

Comment:
For code readability, you should invert the two lines above so you don't need to access ```containersInTabSelected[i]``` twice.